docs/static/setup.sh

```diff
@@ -1,14 +1,25 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
While you are at it, can you please add a header here too after the magic start line?